### ISSUE TITLE
Dragging on desktop "grabs" the image.

### ISSUE THREAD

**Issue by Beanow** [FormidableLabs/nuka-carousel#63]
When dragging to slide on desktop, the image is dragged by the browser as well for drag and drop support. Does this on both Firefox and Chromium (Linux). Especially in Firefox's case it makes it tricky to get to another slide this way.

![Firefox](http://ss.oscp.info/2016-03-11_12-06-13.png)

![Chromium](http://ss.oscp.info/2016-03-11_12-07-25.png)

**Comment by Beanow** [comment]
Looking for examples that didn't have this issue.
https://github.com/roeierez/react-infinite-carousel
I can drag without this happening.

**Comment by Beanow** [comment]
Currently I'm applying the following idea which fixes the problem:
http://stackoverflow.com/questions/4211909/disable-dragging-an-image-from-an-html-page

Using an `onDragStart` event handler:
```js
disableDragging(e){
	e.preventDefault();
	return false;
}

// <img src={image} onDragStart={this.disableDragging} />
```

**Comment by drakmail** [comment]
@Beanow are you tried any of solutions [in this SO thread](http://stackoverflow.com/questions/7439042/css-js-to-prevent-dragging-of-ghost-image)?

**Comment by Beanow** [comment]
I haven't extensively tried different solutions. We've been running the above event handler and it works well on firefox and chromium. With no side effects found on ios and android. We haven't tested on all browsers since we're not targeting them.

It seems that using the `draggable="false"` attribute does go a long way, but selects the dragged image giving an ugly overlay. Using the `user-select:none` style doesn't seem to fix this, in fact it sometimes makes the images draggable again!

The `user-drag:none` style does work, but personally I think CSS is the worst possible place to define behaviour rather than styling.

**Comment by sarmeyer** [comment]
this has been fixed with PR #397
